src/NovelRT.Interop/Graphics/NrtFontSet.cpp

```diff
@@ -21,7 +21,15 @@ extern "C" {
     }
 
     FontSet* fontSetPtr = reinterpret_cast<FontSet*>(fontSet);
-    fontSetPtr->loadFontAsTextureSet(std::string(file), fontSize);
+
+    try
+    {
+      fontSetPtr->loadFontAsTextureSet(std::string(file), fontSize);
+    }
+    catch (const Exceptions::InitialisationFailureException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message? Also the code being used here doesn't match the exception being caught?